docs/_sass/theme/mixins.scss

```diff
@@ -0,0 +1,16 @@
+// Mixins
+// ========================================================================
+//.uk-text-muted a {
```

Comment:
Is the commented out code here serving a purpose?

Reply:
I was a left over comment, thanks for spotting it!